project-base/storefront/components/Pages/BlogCategory/BlogCategoryPageSkeleton.tsx

```diff
@@ -9,7 +9,7 @@ export const BlogCategoryPageSkeleton: FC = () => (
         <div className="mb-16 flex flex-col gap-8 vl:flex-row">
             <div className="flex w-full flex-col gap-20">
                 <div className="flex flex-col justify-center gap-10">
-                    {createEmptyArray(4).map((_, index) => (
+                    {createEmptyArray(9).map((_, index) => (
```

Comment:
Another place :thinking: could this be from a global config?

Reply:
Refactored.